.azure-pipelines/PipelineSteps/test-steps.ps1

```diff
@@ -54,11 +55,13 @@ $ErrorActionPreference = $preference
 Set-Location $currentPath
 Write-Host -ForegroundColor DarkGreen "-------------------- End testing AutoGen modules with PowerShell Core ... --------------------`n`n`n`n`n"
 
-# Analyze test coverage
-Write-Host -ForegroundColor Green "-------------------- Start analyzing test coverage ... --------------------"
-$validateTestCoverageScriptPath = Join-Path $RepoRoot 'tools' 'TestFx' 'Coverage' 'ValidateTestCoverage.ps1'
-& $validateTestCoverageScriptPath
-Write-Host -ForegroundColor DarkGreen "-------------------- End analyzing test coverage ... --------------------`n`n`n`n`n"
+if (-Not $DisableTestCoverage) {
+    # Analyze test coverage
+    Write-Host -ForegroundColor Green "-------------------- Start analyzing test coverage ... --------------------"
+    $validateTestCoverageScriptPath = Join-Path $RepoRoot 'tools' 'TestFx' 'Coverage' 'ValidateTestCoverage.ps1'
+    & $validateTestCoverageScriptPath
+    Write-Host -ForegroundColor DarkGreen "-------------------- End analyzing test coverage ... --------------------`n`n`n`n`n"
+}
 
 # Check test status
```

Comment:
Only the step of analyzing is skipped. Can you explain why checking test status is not skipped as well?

Reply:
nice catch, skip as well